src/agent/onefuzz-agent/src/validations.rs

```diff
@@ -64,10 +64,11 @@ async fn validate_libfuzzer(config: ValidationConfig) -> Result<()> {
     let libfuzzer = LibFuzzer::new(
         &config.target_exe,
         config.target_options.clone(),
-        config.target_env.into_iter().collect(),
+        config.target_env.iter().cloned().collect(),
         config
             .setup_folder
-            .unwrap_or(config.target_exe.parent().unwrap().to_path_buf()),
+            .clone()
+            .unwrap_or_else(|| config.target_exe.parent().unwrap().to_path_buf()),
```

Comment:
Can we use `expect` instead of `unwrap` here to provide a message in case this ever fails?